docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+---
+title: "Creating XML Trees - .NET"
+ms.date: 12/01/2019
+ms.assetid: cc74234a-0bac-4327-9c8c-5a2ead15b595
+---
+
+# Creating XML trees (C# and Visual Basic)
+This topic provides information about creating XML trees in C# and Visual Basic.
+
+## Creating XML trees in C&num;
+This section provides information about creating XML trees in C#.  
+  
+For information about using the results of LINQ queries as the content for an <xref:System.Xml.Linq.XElement>, see [Functional Construction (LINQ to XML)](./functional-construction-linq-to-xml.md).  
+  
+### Constructing elements
+The signatures of the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XAttribute> constructors let you pass the contents of the element or attribute as arguments to the constructor. Because one of the constructors takes a variable number of arguments, you can pass any number of child elements. Of course, each of those child elements can contain their own child elements. For any element, you can add any number of attributes.  
```

Comment:
```suggestion

The signatures of the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XAttribute> constructors let you pass the contents of the element or attribute as arguments to the constructor. Because one of the constructors takes a variable number of arguments, you can pass any number of child elements. Of course, each of those child elements can contain their own child elements. For any element, you can add any number of attributes.  
```

Reply:
Maira I don't understand this suggested change. I see that "signatures" is highlighted, but I don't see why. And I see a new blank line, but we are not putting blank lines after headings. Please advise.

Reply:
The suggestion feature doesn't render Markdown well.